DigitalLearningSolutions.Data/Services/CourseDelegatesDownloadFileService.cs

```diff
@@ -190,5 +187,32 @@ CourseAdminFields adminFields
 
             dataTable.Rows.Add(row);
         }
+
+        private static void FormatWorksheetColumns(IXLWorkbook workbook, DataTable dataTable)
+        {
+            var dateColumns = new[] { Enrolled, LastAccessed, CompleteBy, CompletedDate, RemovedDate };
+            foreach (var columnName in dateColumns)
+            {
+                var columnIndex = dataTable.Columns.IndexOf(columnName) + 1;

```

Comment:
Should this (and ones in following loops) be named "columnNumber" given we are using `Columns.IndexOf()` and adding 1 to it?